src/apps/btc/btc_common.c

```diff
@@ -165,28 +165,33 @@ bool btc_common_format_amount(uint64_t satoshi, char* out, size_t out_len)
     char remainder_str[21] = {0};
     char* remainder_str_start = NULL;
     _sprint_uint64(remainder, remainder_str, &remainder_str_start);
+    char amount_str[30] = {0};
     int sprintf_result = snprintf(
-        out,
-        out_len,
+        amount_str,
+        sizeof(amount_str),
         "%s.%.*s%s",
         quotient_str_start,
         // pad up to 8 zeroes
```

Comment:
Could the comment say `// left pad 'remainder_str' up to 8 zeros`?

Reply:
Done